ceph_ansible.yml

```diff
@@ -6,7 +6,10 @@
       setup:
 
 - name: Install ceph-ansible and install Ceph
-  hosts: qe_server
-  user: root
+  hosts: localhost
+  connection: local
+  become: yes
+  become_user: root
+  become_method: sudo
```

Comment:
It is not necessary (and recommended) to run `ceph-ansible` role as `root`, it should be executed under the same user who launch the whole playbook. So the above three lines (11-13) are should not be there.